app/assets/stylesheets/list.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the List controller here.
```

Comment:
Not sure if this ought to be committed until the file has actual contents. The stub doesn't really hurt anything, but it adds clutter, particularly if the list object doesn't wind up having its own style sheets.